NEWS.md

```diff
@@ -1,5 +1,11 @@
 # What's New
 
+## Update June 21, 2021
+
+We have a new release [Recommenders 0.6.0](https://github.com/microsoft/recommenders/releases/tag/0.6.0)!
+
+Recommenders is now on PyPI and can be installed using pip! In addition there are lots of bug fixes and utilities improvements.
```

Comment:
Do we want to provide a link here pointing to the details about the pip installable package? Maybe the document as well - it is really helpful a lot of times people can easily reference the API doc.

Reply:
yeah that's a good idea! I'll make some changes